apps/passport-client/components/core/Chip.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import styled from "styled-components";
+
+/**
+ * Chips are small, compact elements that represent an attribute, text, or action.
+ *
+ * See https://m3.material.io/components/chips/overview
+ */
+const Chip: React.FC<{
```

Comment:
could you declare this functional component in a way consistent with the way the rest of them are defined in this project?

```
export function Chip({...}) ...
```